### PR TITLE
Update vpc peering doc 20240706

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 1]
```diff
@@ -67,18 +67,22 @@ This section describes how to set up VPC peering connections on AWS. For Google
 
     The **VPC Peering** configuration is displayed by default.
 
-4. Click **Add**, choose the AWS icon, choose the **TiDB Cloud VPC Region**, and then fill in the required information of your existing AWS VPC:
+4. Click **Create VPC Peering**, choose **AWS VPC Peering**, choose the **TiDB Cloud VPC Region**, and then fill in the required information of your existing AWS VPC:
 
     - Your existing VPC Region
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:72]:
```suggestion
    - Your VPC Region
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 2]
```diff
@@ -67,18 +67,22 @@ This section describes how to set up VPC peering connections on AWS. For Google
...
     - AWS Account ID
     - VPC ID
     - VPC CIDR
 
-    You can get these information from your VPC details on the VPC dashboard. TiDB Cloud supports creating VPC peerings between VPCs in the same region or from two different regions.
+    You can get these information from your VPC details page on the AWS console. TiDB Cloud supports creating VPC peerings between VPCs in the same region or from two different regions.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:77]:
```suggestion
    You can get such information from your VPC details page on the [AWS Management Console](https://aws.amazon.com/console). TiDB Cloud supports creating VPC peerings between VPCs in the same region or from two different regions.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 3]
```diff
@@ -67,18 +67,22 @@ This section describes how to set up VPC peering connections on AWS. For Google
...
 
     ![VPC peering](/media/tidb-cloud/vpc-peering/vpc-peering-creating-infos.png)
 
-5. Click **Initialize**. The **Approve VPC Peerings** dialog is displayed.
+5. Click **create**. The VPC peering request is sent and you can see the VPC Peering info in **VPC Peering** tab and the status is "System Checking".
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:81]:
```suggestion
5. Click **Create** to send the VPC peering request, and then view the VPC peering information on the **VPC Peering** tab. The status of the newly created VPC peering is **System Checking**.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 4]
```diff
@@ -67,18 +67,22 @@ This section describes how to set up VPC peering connections on AWS. For Google
...
+
+6. Check **...** of the VPC Peering record, and click **View** , you can view the connection information of your TiDB VPC peerings from **VPC Peering Details** page.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:83]:
```suggestion
6. To view detailed information about your newly created VPC peering, click **...** > **View**  in the **Action** column. The **VPC Peering Details** page is displayed.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 5]
```diff
@@ -223,9 +227,9 @@ Now you have successfully set up the VPC peering connection. Next, [connect to t
     - VPC Network Name
     - VPC CIDR
 
-5. Click **Initialize**. The **Approve VPC Peerings** dialog is displayed.
+5.  Click **create**. The VPC peering request is sent and you can see the VPC Peering info in **VPC Peering** tab and the status is "System Checking".
 
-6. Check the connection information of your TiDB VPC peerings.
+6. Check **...** of the VPC Peering record, and click **View** , you can view the connection information of your TiDB VPC peerings from **VPC Peering Details** page.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:232]:
```suggestion
6. To view detailed information about your newly created VPC peering, click **...** > **View**  in the **Action** column. The **VPC Peering Details** page is displayed.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 6]
```diff
@@ -223,9 +227,9 @@ Now you have successfully set up the VPC peering connection. Next, [connect to t
     - VPC Network Name
     - VPC CIDR
 
-5. Click **Initialize**. The **Approve VPC Peerings** dialog is displayed.
+5.  Click **create**. The VPC peering request is sent and you can see the VPC Peering info in **VPC Peering** tab and the status is "System Checking".
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:230]:
```suggestion
5. Click **Create** to send the VPC peering request, and then view the VPC peering information on the **VPC Peering** tab. The status of the newly created VPC peering is **System Checking**.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 7]
```diff
@@ -175,7 +179,7 @@ You can also use the AWS dashboard to configure the VPC peering connection.
 
     2. From the left navigation bar, open the **Peering Connections** page. On the **Create Peering Connection** tab, a peering connection is in the **Pending Acceptance** status.
 
-    3. Confirm the requester owner is TiDB Cloud (`380838443567`，`143458967504`, or `730335266318`). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
+    3. Confirm the requester owner is TiDB Cloud (You can view the "TiDB Cloud AWS Account ID" and "TiDB Cloud VPC ID" from **VPC Peering Details** page on [TiDB Cloud console](https://tidbcloud.com)). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:182]:
```suggestion
    3. Confirm that the requester owner and the requester VPC match **TiDB Cloud AWS Account ID** and **TiDB Cloud VPC ID** on the **VPC Peering Details** page of the [TiDB Cloud console](https://tidbcloud.com). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```

File: tidb-cloud/set-up-vpc-peering-connections.md (hunk 1 above)
Comment (qiancai) [reply tidb-cloud/set-up-vpc-peering-connections.md:72]: image

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 8]
```diff
@@ -223,9 +227,9 @@ Now you have successfully set up the VPC peering connection. Next, [connect to t
     - VPC Network Name
     - VPC CIDR
```
Comment (hfxsd) [inline tidb-cloud/set-up-vpc-peering-connections.md:229]: Change Application Google Cloud Project ID to Google Cloud Project ID
ededa1a8-6aa8-4a19-a7f2-9aca707e5903

Comment (qiancai) [reply tidb-cloud/set-up-vpc-peering-connections.md:229]: Done: https://github.com/pingcap/docs/pull/18149/commits/43eb51e46463530a033a410fe1863bbb86cbca8d

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 9]
```diff
@@ -213,19 +217,19 @@ Now you have successfully set up the VPC peering connection. Next, [connect to t
...
-    - Application Google Cloud Project ID
+    - Google Cloud Project ID
     - VPC Network Name
     - VPC CIDR
 
-5. Click **Initialize**. The **Approve VPC Peerings** dialog is displayed.
+5. Click **Create** to send the VPC peering request, and then view the VPC peering information on the **VPC Peering** tab. The status of the newly created VPC peering is **System Checking**.
 
-6. Check the connection information of your TiDB VPC peerings.
+6. To view detailed information about your newly created VPC peering, click **...** > **View**  in the **Action** column. The **VPC Peering Details** page is displayed.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:232]:
```suggestion
6. To view detailed information about your newly created VPC peering, click **...** > **View** in the **Action** column. The **VPC Peering Details** page is displayed.
```

**Review by Yuiham — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**